packages/sample-app/sample_app/quickstart_openai.py

```
@@ -0,0 +1,39 @@
+        )
+        
+        # Print the response out to the console
+        print("\n🤖 AI Response:")
+        print(response.choices[0].message.content)
+        print("\n✅ Trace successfully captured! Check your Traceloop dashboard.")
+
+    except Exception as e:
+        print(f"❌ An error occurred: {e}")
+
+if __name__ == "__main__":
+    main()
```

Ruff (0.15.12)

[warning] No newline at end of file

Add trailing newline

(W292)